pkg/controller/memberstatus/memberstatus_controller.go

```diff
@@ -42,6 +42,18 @@ const (
 	defaultRequeueTime      = time.Second * 5
 )
 
+// custom component condition error messages & reasons
+const (
+	// deployment not found
+	reasonNoDeployment         = "DeploymentNotFound"
+	errMsgNoMemberOperatorName = "unable to look up the member operator name, the environment variable OPERATOR_NAME is not set"
+	errMsgCannotGetDeployment  = "unable to get the member operator deployment"
+
+	// kubefed not found
+	reasonHostConnectionNotFound = "KubefedNotFound"
+	errMsgHostConnectionNotFound = "the host connection was not found"
```

Comment:
As mentioned above, let's not use these constants in tests. It's OK to keep using them in the controller of course.